docs/content/preview/yugabyte-platform/troubleshoot/install-upgrade-issues/kubernetes.md

```diff
@@ -7,283 +7,284 @@ menu:
+- To resolve the Bad pull secret, No pull secret, Bad pull secret name errors, enable the pull secret to fetch the images from YugabyteDB Quay.io registry and ensure that you have applied the same in the namespace that will be used to install YugabyteDB Anywhere. By default, the search for a secret with name `yugabyte-k8s-pull-secret` is performed. For more information, see [values.yaml](https://github.com/yugabyte/charts/blob/24a8dcf3a4c33153477e3e3ba82f9f4b6e2967a5/stable/yugaware/values.yaml#L16).
+- To resolve the Unable to pull image error, ensure that the Kubernetes nodes can connect to Quay.io or you have images in the local registry. For more information, see [Pull and push yugabytedb docker images to private container registry](../../../install-yugabyte-platform/prepare-environment/kubernetes/#pull-and-push-yugabytedb-docker-images-to-private-container-registry).
 
 
-### Pods are in CrashLoopBackOff
+### CrashLoopBackOff error
 
-**Symptom**
+There is a number of reasons for the `CrashLoopBackOff` error. It typically occurs when a YugabyteDB Anywhere pod crashes due to an internal application error.
 
-There can be multiple reasons behind `CrashLoopBackOff`. YBA pod can crash due to some internal application error.
+To start diagnostics, execute the following command to obtain the pod:
 
 ```sh
 kubectl get pod -n <NAMESPACE>
 ```
+If the issue you are experiencing is due to the `CrashLoopBackOff` error, expect to see this error listed under `STATUS`, as per the following output:
 
-```sh
+```output
 NAME                             READY   STATUS             RESTARTS   AGE
 yugabyte-platform-1-yugaware-0   3/4     CrashLoopBackOff   2          4d14h
 ```
 
-**Debug**
-
-You can use `kubectl` to analyze the situation. We need to check YBA logs to understand the failure.
+**Resolution**
 
-- Describe the YBA pod facing `CrashLoopBackOff`. Check for `Events`.
+- Execute the following command to obtain detailed information about the YugabyteDB Anywhere pod experiencing the `CrashLoopBackOff` error:
 
   ```sh
   kubectl describe pods <POD_NAME> -n <NAMESPACE>
   ```
 
-- Check the logs for particular container
+- Execute the following commands to check YugabyteDB Anywhere logs for a specific container and perform troubleshooting based on the information in the logs:
 
   ```sh
-  # yugabytedb anywhere
+  # YugabyteDB Anywhere
   kubectl logs <POD_NAME> -n <NAMESPACE> -c yugaware
 
-  # postgres
+  # PostgreSQL
   kubectl logs <POD_NAME> -n <NAMESPACE> -c postgres
   ```
 
-## Running but load balancer service is not ready
+## Load balancer service is not ready on a running universe
```

Comment:
We should remove the `on a running universe` because this section refers to LB to serve YugabyteDB Anywhere, not the YugabyteDB universe.

Reply:
Changed